common-lib/src/main/java/com/brandwatch/minibcr/common/CommonApplication.java

```diff
@@ -1,11 +1,8 @@
 package com.brandwatch.minibcr.common;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-
-@SpringBootApplication
 public class CommonApplication {
-    public static void main(String[] args) {
-        SpringApplication.run(CommonApplication.class);
-    }
+
+  public static void main(String[] args) {
```

Comment:
What's happening here?

Reply:
This class is unnecessary which is removed in the first PR. Once merged this class will be deleted.

Reply:
Ah I see. Going forward, it would be good to pay attention to having cleaner PRs where something like this isn't there. Too many PR's open at the same time can cause these funny cross dependencies which shouldn't really exist. I noticed a few other things like that in your PRs, for example when you touch the pom files of unrelated services.